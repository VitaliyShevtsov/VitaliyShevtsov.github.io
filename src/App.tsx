import './App.css';
import { Toaster } from './components/ui/toaster';
import AddressLookup from './features/AddressLookup';

function App() {
  return (
    <>
      <AddressLookup />
      <Toaster />
    </>
  );
}

export default App;
