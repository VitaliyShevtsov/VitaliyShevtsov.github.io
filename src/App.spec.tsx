import { componentTestBed } from '@/test';
import App from './App';

describe('<App />', () => {
  it('should match snapshot', () => {
    const { container } = componentTestBed(<App />);

    expect(container).toMatchSnapshot();
  });
});
