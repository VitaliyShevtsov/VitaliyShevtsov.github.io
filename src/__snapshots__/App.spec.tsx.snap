// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<App /> > should match snapshot 1`] = `
<div>
  <script
    nonce=""
  >
    ((e,i,s,u,m,a,l,h)=&gt;{let d=document.documentElement,w=["light","dark"];function p(n){(Array.isArray(e)?e:[e]).forEach(y=&gt;{let k=y==="class",S=k&&a?m.map(f=&gt;a[f]||f):m;k?(d.classList.remove(...S),d.classList.add(a&&a[n]?a[n]:n)):d.setAttribute(y,n)}),R(n)}function R(n){h&&w.includes(n)&&(d.style.colorScheme=n)}function c(){return window.matchMedia("(prefers-color-scheme: dark)").matches?"dark":"light"}if(u)p(u);else try{let n=localStorage.getItem(i)||s,y=l&&n==="system"?c():n;p(y)}catch(n){}})("class","theme","system",null,["light","dark"],null,true,true)
  </script>
  <div
    class="css-0"
  >
    <div
      class="css-82qlwu"
    >
      <h2
        class="chakra-heading css-eprwor"
      >
        Address Lookup Feature
      </h2>
      <p
        class="css-0"
      >
        Enter one or more IP addresses and get their country
      </p>
      <button
        class="chakra-button css-qeumhc"
        type="button"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 1024 1024"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M482 152h60q8 0 8 8v704q0 8-8 8h-60q-8 0-8-8V160q0-8 8-8Z"
          />
          <path
            d="M192 474h672q8 0 8 8v60q0 8-8 8H160q-8 0-8-8v-60q0-8 8-8Z"
          />
        </svg>
         Add
      </button>
    </div>
  </div>
</div>
`;
